src/spike_source.hpp

```diff
@@ -12,7 +12,8 @@ class spike_detector {
 public:
     using cell_type = Cell;
 
-    spike_detector(const cell_type& cell, typename Cell::detector_handle h, double thresh, float t_init) :
+    spike_detector(
+        const cell_type& cell, typename Cell::detector_handle h, double thresh, float t_init) :
```

Comment:
Parameters line up with initializers, without any intervening punctuation. I think it's a confusing compromise. We can nut this out on our coding guidelines bike shed channel.